lightcurvedb/storage/base/schema.py

```diff
@@ -0,0 +1,57 @@
+"""
+Base SQL schema definitions.
+"""
+
+
+
+SOURCES_TABLE = """
+CREATE TABLE IF NOT EXISTS sources (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
```

Comment:
`VARCHAR` confers [no performance improvement over TEXT](https://www.postgresql.org/docs/current/datatype-character.html) and is more limiting.